### ISSUE TITLE
Latest version causes issues when running in browser with ngCordova

### ISSUE THREAD

**Issue by PyriteParrot** [driftyco/ionic-plugin-keyboard#178]
We've been using version 1.0.6 of the Keyboard plugin and have had no issues when running our app in the browser (with Cordova plugins loaded) in conjunction with ngCordova. We recently upgraded to the latest plugin version, 1.0.8, and now we are seeing exceptions thrown from ngCordova that cause the app to break when run in the browser. The following error gets thrown during initialization of ngCordova: 

`TypeError: Cannot read property 'Keyboard' of undefined
    at null.<anonymous> (http://localhost:8100/lib/ngCordova/dist/ng-cordova.js:4611:26)`

Any subsequent calls using the $cordovaKeyboard service also throw exceptions as `cordova.plugins.Keyboard` is undefined.

The issue seems to stem from [this change](https://github.com/driftyco/ionic-plugin-keyboard/commit/b460a7219c7272ad9f086d9adc012f047dd7687c), where specialized JS modules were created for Android and iOS. This causes the browser platform not to load any keyboard.js module, which leaves `cordova.plugins.Keyboard` undefined, breaking ngCordova in the browser. I understand that the keyboard plugin isn't supported for the browser platform, but perhaps there should be a simple proxy object that gets loaded for the browser?

Is there any way to work around this? For now we've simply downgraded the plugin to 1.0.6 as we make heavy use of the browser platform for testing.

**Comment by PyriteParrot** [comment]
We ended up running into issue #102 so we had to go back to version 1.0.8. I added the following to `plugin.xml` and this allows the current version to function as it did in 1.0.6 when run in the browser:

```
<!-- browser -->
<platform name="browser">
    <js-module src="www/android/keyboard.js" name="keyboard">
        <runs/>
        <clobbers target="cordova.plugins.Keyboard" />
    </js-module>
</platform>
```

**Comment by tlancina** [comment]
Yes we should support the browser platform and mocks the calls, or at least have the methods be a noop so they don't error.

**Comment by theaccordance** [comment]
This is a pretty big issue - it completely breaks browser emulation if 'cordova.plugins.Keyboard' is referenced during early enough in the app's initialization (in our case, we begin calling methods in the run block).  Based on the patch method described above, I've submitted #190 as a quick-fix PR.  It provides a basic mock to prevent the error.

Take a look when you get a chance, let me know if I need to adjust anything to meet your code standards.  Just wanna get this issue resolved sooner rather than later so my team doesn't have to patch the plugin every time they reload their local plugins.

**Comment by theaccordance** [comment]
It's been a month since I submitted the PR... any update on when we might see this resolved and thrown into a release?

**Comment by tlancina** [comment]
Thanks for the ping and your patience! Just released 2.1.0 with your PR, please let me know if you have any issues.
